### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/dependencies/sub-dependencies.md`

### PR DESCRIPTION
**Pull request by Joao-Pedro-P-Holanda**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 208f21aba1dc97ff466a430fb6f98bc4b6a8ce29 at: https://7437920d.fastapitiangolo.pages.dev

File: docs/pt/docs/tutorial/dependencies/sub-dependencies.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    ```Python hl_lines="13"
+    {!> ../../../docs_src/dependencies/tutorial005.py!}
+    ```
+
+Vamos focar nos parâmetros declarados:
+
+* Mesmo que essa função seja uma dependência ("injetável") por si mesma, ela também declara uma outra dependência (ela "depende" de outra coisa).
+    * Ela depende do `query_extractor`, e atribui o valor retornado pela função ao parâmetro `q`.
+* Ela também declara um cookie opcional `last_query`, do tipo `str`.
+    * Se o usuário não passou nenhuma consulta `q`, a última consulta é utilizada, que foi salvada em um cookie anteriormente.
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/dependencies/sub-dependencies.md:98]: although _salvada_ is right, in this case _que foi salva em um cookie_ sounds better.

but it's not a mistake

**Review by ceb10n — APPROVED** [review]
Great translation! 🚀

File: docs/pt/docs/tutorial/dependencies/sub-dependencies.md (hunk 1 above)
Comment (ceb10n) [reply docs/pt/docs/tutorial/dependencies/sub-dependencies.md:98]: I would translate like:

_a última consulta que foi salva anteriormente em um cookie é utilizada._

Comment (Joao-Pedro-P-Holanda) [reply docs/pt/docs/tutorial/dependencies/sub-dependencies.md]: I also think that "foi salvada" seems better, primarly for non-brazilian portuguese speakers, thanks for spotting it!

**Comment by tiangolo** [comment]
📝 Docs preview for commit 41032464a0831db6d78cec542d3d85dbf0f0d564 at: https://fe755612.fastapitiangolo.pages.dev

**Review by ceb10n — APPROVED** [review]

**Comment by alejsdev** [comment]
Cool, thanks! @Joao-Pedro-P-Holanda @ceb10n  :sunglasses: :v:

**Pull request merged**